central/complianceoperator/v2/benchmarks/datastore/datastore_impl.go

```diff
@@ -0,0 +1,26 @@
+package datastore
+
+import (
+	"context"
+
+	"github.com/stackrox/rox/central/complianceoperator/v2/benchmarks/store/postgres"
+	"github.com/stackrox/rox/generated/storage"
+)
+
+var _ DataStore = (*datastoreImpl)(nil)
```

Comment:
+1, i like it :)

Reply:
This is forcing the compiler to evaluate whether the `datastoreImpl` implements the interface `DataStore`. Since it is getting assigned to the blank-identifier (`_`) this should be discarded after compiling.